Microsoft/We.yml

```diff
@@ -4,4 +4,8 @@ link: https://docs.microsoft.com/en-us/style-guide/grammar/person#avoid-first-pe
 level: warning
 ignorecase: true
 tokens:
-  - we
+  - \bwe\b
```

Comment:
You should be able to drop the word boundaries for all of these tokens. The only reason they're needed in `FirstPerson.yml` is because I had to set `nonword: true` for the `I` workaround stuff.

Reply:
ah.. k I'll fix